index.html

```diff
@@ -561,8 +561,8 @@ <h3>The <dfn><code>pointercancel</code> event</dfn></h3>
                 <ul>
                     <li>The user agent has determined that a pointer is unlikely to continue to produce events (for example, because of a hardware event).</li>
                     <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).
-                     <div class="note">User agents can use multiple pointers for a particular action like scrolling or zooming.
-                     In that case they can cancel all the pointers they deem necessary.</div></li>
+                     <div class="note">User agents can trigger panning or zooming through multiple pointer types (say "touch" and "pen"),
+                     and therefore can possibly cancel all pointers of those types for a single viewport manipulation.</div</li>
```

Comment:
i'd reword the whole "and therefore...manipulation" to "and therefore the start of a pan or zoom action may result in the cancellation of various pointers, including pointers with different pointer types." or similar. Seems a bit clearer (assuming I understood this right)